desktop/renderer/style.css

```diff
@@ -106,6 +106,7 @@ table {
 
 html {
   box-sizing: border-box;
+  cursor: default;
```

Comment:
our text style uses cursor: inherit by default. this is cause we set cursor on boxes and things so we want that to inherit down. somehow the default root changed or has been broken such that most of the text in the app had the 'text'/ selectable cursor. instead i'm setting the html root style to use default so all text in our app by default will not look selectable (since its an app and not a webpage)